app/src/NativeBase/Theme/StaTheme.tsx

```diff
@@ -158,9 +186,20 @@ const StaTheme = extendTheme({
         }),
       }),
     },
+    Input: {
+      baseStyle: { fontFamily: 'primary' },
+      defaultProps: {
+        borderWidth: '1',
+        borderColor: 'grey.60',
+        marginX: '2.5',
```

Comment:
Same point as above for buttons -- don't think they should have a default X margin as they look like they should usually fill the available width of the container they're in, so horizontal spacing is better handled by the container (e.g. `ScrollView` or `Box`)